### ISSUE TITLE
Render In R Markdown with Shiny

### ISSUE THREAD

**Issue by dcmg21** [hafen/trelliscopejs#127]
Is there any instructions with the best way to render trelliscope in a markdown file?  I can render this fine in R Studio but cannot get this to workin in a markdown file using runtime shiny.

**Comment by hafen** [comment]
I've been working on a new version of Trelliscope and have put some extra focus on embedding. If you'd like to try this out, please take a look here: https://trelliscope.org/trelliscope/articles/trelliscope.html and here: https://trelliscope.org/trelliscope/articles/embed.html.

**Comment by dcmg21** [comment]
Oh awesome!  Thanks for the reply.  I finally got it implemented in my shiny app but it took forever.  Instructions were not very clear.  Is there a difference between trelliscopeJS and trelliscope?  Thanks again!

**Comment by hafen** [comment]
Yeah sorry for lack of response! There are many differences but the principles are all the same. Both the R and JS libraries have been completely re-written and I've changed things to reflect workflows I've been wanting to support. Hopefully a short learning curve and it should be worth it as this package is where efforts will be focused going forward.

**Comment by dcmg21** [comment]
Oh awesome!  No worries man I'm sure you are busy.  So going forward I should look at trelliscope and not trelliscopeJS?

**Comment by hafen** [comment]
Yes I would recommend that. Just note that it hasn't seen widespread use yet so let me know if you have any issues or feedback.

**Comment by dcmg21** [comment]
Will do man.  I always look for tools to use in business and this one has serious potential.  Keep it up!
